src/platform/packages/shared/kbn-default-tracer/index.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { trace } from '@opentelemetry/api';
+
+export function getDefaultTracer() {
+  return trace.getTracer('kibana');
```

Comment:
After having played with this a bit more, I'm not so convinced that we want to reuse a default tracer.

I think that having scoped tracers is better (maybe I'm biased by how it works in metrics, though).

Not a blocker... just a thought.

`kibana` is already at the `resource.attributes.service.name`.

Reply:
separate tracers are generally for different instrumentations. I don't think there's a lot of usefulness in using separate tracers in Kibana, especially as there is no clear hierarchy as you would see with instrumentations which are always at the tail of a dependency graph.

Reply:
I think we can just set `transaction.type` as an attribute: https://github.com/elastic/kibana/blob/cbd3a97eb241e09e63049842518df83b1c64bebd/src/platform/packages/shared/kbn-opentelemetry-attributes/src/attributes/tracing.ts#L11